### DISCUSSION TITLE
installing mysql client on redhat linux 8 with apache

### DISCUSSION THREAD

**Discussion by bneal1987** [apache/airflow#21771] [Q&A]
Working on try to install the apache-airflow-providers-mysql and running into an error when trying to install dependency for mysql client. 

We are currently running Postgres and would like to only install the mysql client to meet the requirements to allow us to install the provider package. Need the provider package so we can create a connection to our Informatica database.

We were able to install the mysql connector without any issues, but not the mysql client. 

According to apache documents we have the proper version of Airflow and connector, but unable to install the mysql client. 

I am attaching the errors because I am not very good with understanding logs within linux or MySQL errors. 

Looking for any feedback that you may have.

Thanks,
[Uploading mysql client error 2-23-2022.docx…]()

**Comment by potiuk** [comment]
You have not attached the logs. And please try to analyze the logs and figure what is the errors  - this will increase your likelihood of somoene attempting to help you - the more you narrow your case on your own, the better.
